docs/introduction/01-setting-up.mdx

```diff
@@ -24,48 +23,41 @@ node -v
+## Creating a new project
 
-The fastest and best way to create new projects with Crawlee is to use the [Crawlee CLI](https://www.npmjs.com/package/@crawlee/cli). This command line tool allows you to create and run Crawlee projects with ease. You can use the `npx` utility to download and run the CLI - it is also embedded to the `crawlee` meta-package:
+The fastest and best way to create new projects with Crawlee is to use the [Crawlee CLI](https://www.npmjs.com/package/@crawlee/cli). You can use the `npx` utility to download and run the CLI - it is also embedded in the `crawlee` package:
 
 ```bash
 npx crawlee create my-new-project
 ```
 
-A prompt will be shown, asking you to choose a template. Let's choose the first one called `Crawlee playwright template [TypeScript]`. The command will now create a new directory in your current working directory, called `my-new-project`, create a `package.json` in this folder and install all the necessary dependencies. It will also add example source code that you can immediately run.
+A prompt will be shown, asking you to choose a template. Crawlee is written in TypeScript so if you're familiar with it, choosing a TypeScript template will give you better code completion and static type checking, but feel free to use JavaScript as well. Functionally they're identical.
+
+Let's choose the first template called **Crawlee playwright template**. The command will create a new directory in your current working directory, called **my-new-project**, add a **package.json** to this folder and install all the necessary dependencies. It will also add example source code that you can immediately run.
 
 Let's try that!
 
 ```bash
 cd my-new-project
+npm start
 ```
 
-```bash
-npx crawlee run
-```
-
-We used the `crawlee run` CLI command, but all what it actually does is to call `npm run start`. You can use the `--script` or `-s` option to specify what script you want to run instead of the default `start`:
-
-```bash
-npx crawlee run --script=start:prod
-```
-
-:::info Purging of storages
+You will see log messages in the terminal as Crawlee boots up and after a second a Chromium browser window will open. In the window, you'll see quickly changing pages and back in the terminal, you will see the printed titles (contents of the `<title>` HTML tags) of the pages.
 
-Another option of the `crawlee run` command is the control over purging: `--purge` and more importantly `--no-purge`, as purging is enabled by default. To understand what those are about, you first need to know about the concept of storages in Crawlee. There are 3 storage types you can use: <ApiLink to="core/class/RequestQueue">`RequestQueue`</ApiLink>, <ApiLink to="core/class/KeyValueStore">`KeyValueStore`</ApiLink> and <ApiLink to="core/class/Dataset">`Dataset`</ApiLink>. Crawlee is using the first two for storing the runtime data about current crawl, and the third one is for storing the results.
+![Chrome Scrape](/img/chrome_scrape.gif)
```

Comment:
It's a battle of file size and quality. A big one had well over 1 MB :( But we can settle this later, because we will have to do this again with the new design of the website anyway 😄